src/components/DateTimePicker/DateTimePicker.test.jsx

```diff
@@ -348,6 +348,61 @@ describe('DateTimePicker', () => {
     ).toHaveLength(1);
   });
 
+  it('should render with programmatically set absolute range', () => {
+    const wrapper = mount(<DateTimePicker {...dateTimePickerProps} />);
```

Comment:
Since this is a sev 1 bug fix I am not going to hold up the PR but in the future we want to utilize the react testing library for any new test.